packages/gatsby-source-strapi/src/gatsby-node.js

```diff
@@ -0,0 +1,177 @@
+import { fetchStrapiContentTypes, fetchEntities, fetchEntity } from './fetch';
+import { downloadMediaFiles } from './download-media-files';
+import {
+  buildMapFromNodes,
+  buildNodesToRemoveMap,
+  getEndpoints,
+  makeParentNodeName,
+} from './helpers';
+import { createNodes } from './normalize';
+
+const LAST_FETCHED_KEY = 'timestamp';
+
+exports.onPreInit = () => console.log('Loaded gatsby-source-strapi-plugin');
+
+exports.sourceNodes = async (
```

Comment:
I think eslint will request this, but you'll need to update these to esm exports.